cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -908,6 +913,67 @@ public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, Tie
         return this;
     }
 
+    /**
+     * Configures the quotas based on the type of the plugin - {@link QuotasPluginKafka}, {@link QuotasPluginStrimzi}
+     *
+     * @param clusterName    Name of the cluster
+     * @param quotasPlugin   Configuration of the quotas plugin
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withQuotas(String clusterName, QuotasPlugin quotasPlugin) {
+        if (quotasPlugin != null) {
+            printSectionHeader("Quotas configuration");
+
+            // for the built-in Kafka quotas plugin we don't need to configure anything
+            if (quotasPlugin instanceof QuotasPluginStrimzi quotasPluginStrimzi) {
+                configureQuotasPluginStrimzi(clusterName, quotasPluginStrimzi);
+            }
+
+            writer.println();
+        }
+
+        return this;
+    }
+
+    /**
+     * Configures Strimzi quotas plugin
+     *
+     * @param clusterName           Name of the cluster
+     * @param quotasPluginStrimzi   Strimzi quotas plugin configuration
+     */
+    private void configureQuotasPluginStrimzi(String clusterName, QuotasPluginStrimzi quotasPluginStrimzi) {
+        // add TopicOperator's user to the excluded principals
+        List<String> excludedPrincipals = new ArrayList<>(List.of(CruiseControlApiProperties.API_TO_ADMIN_NAME));
```

Comment:
You do not seem to include any Topic OPerator user (makes sense as it doesn't use it). but you list it in the comment. And you add soem Cruise Control user but do not list it either. Does CC actually produce any messages? If it does, `API_TO_ADMIN_NAME` certainly isn't it (check the super.users configuration).

Reply:
Ah okay, I wasn't sure which user should be used here and if it doesn't have some kind of "generated" name. Also I wasn't sure if I should cover the scenario when the TO will be deployed separately (and not part of the Kafka resource deployment in EO) - and if it somehow differs in terms of the users. I will add the correct users there, thanks